styles/mixins/patterns/_containers.scss

```diff
@@ -235,3 +235,26 @@ $-header-height: var.$touch-target-size;
 @mixin scrollbox--with-header {
   @include scrollbox($shadow-top-position: $-header-height);
 }
+
+/**
+ * A sticky container that is sized one-touch-unit high. This is currently
+ * expanded upon by table styling, but not used in other patterns (yet).
+ */
+@mixin sticky-header {
+  position: sticky;
+  top: 0;
+  height: $-header-height;
+
+  @include layout.padding;
+  @include atoms.border(bottom);
+
+  background-color: var.$color-grey-1;
```

Comment:
Maybe the colour could be taken out of this pattern, so the pattern only deals about positioning and size.